### DISCUSSION TITLE
Hello from Japan

### DISCUSSION THREAD

**Discussion by eNipu** [OpenMined/courses#89] [Introduce Yourself]
Hi Everyone, I am Amin, I'm really  passionate about privacy preserving technologies in the context of AI and BigData. Looking forward to learning and growing together.

**Comment by yemikifouly** [comment]
Hello Amin! 👋 Welcome to OpenMined! 🙂 I hope you enjoy the course and find some interesting answers to questions you had and even some you didn't. 🙂
